backend/api/api_deployment_views.py

```diff
@@ -119,9 +125,27 @@ def create(
             headers=headers,
         )
 
-
-def get_error_from_serializer(error_details: dict[str, Any]) -> Optional[str]:
-    error_key = next(iter(error_details))
-    # Get the first error message
-    error_message: str = f"{error_details[error_key][0]} : {error_key}"
-    return error_message
+    @action(detail=True, methods=["get"])
+    def download_postman_collection(
+        self, request: Request, pk: Optional[str] = None
+    ) -> Response:
+        """Custom action to fetch a single instance."""
```

Comment:
@chandrasekharan-zipstack is this doc string relevant to the method? Can you add a proper description?